options/locale/locale_en-US.ini

```diff
@@ -429,6 +430,8 @@ remove_account_link = Remove linked account
 remove_account_link_desc = Delete this account link will remove all related access for your account. Do you want to continue?
 remove_account_link_success = Account link has been removed successfully!
 
+repos_none = You do not owned any repository
```

Comment:
The phrasing on this is a bit awkward, "You do not own any repositories" is more correct. Although I'm not sure using "own" here is the right term either.